src/Loop/Driver.php

```diff
@@ -15,7 +15,14 @@
     private $registry = [];
 
     /**
-     * Start the event loop.
+     * Run the event loop.
+     *
+     * One iteration of the loop is called one "tick". A tick covers the following steps:
+     *
+     *  1. Activate watchers created / enabled in the last tick / before `run`.
+     *  2. Execute all enabled defer watchers.
+     *  3. Execute all due timers, each timer only once per tick.
+     *  4. Execute actionable stream watchers, each only once per tick.
```

Comment:
Uh, there is no order guarantee between timers and stream watchers. Merge these two into a single step.